.github/workflows/codeql.yml

```
@@ -61,3 +61,45 @@ jobs:
       uses: github/codeql-action/analyze@v3
       with:
         category: "/language:${{matrix.language}}"
+
+  pr-size-labeler:
+    name: PR Size Labeler
+    # Only run on pull requests
+    if: github.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{ github.event.pull_request.head.sha }}
+          
+      - name: PR Size Labeler
+        uses: codelytv/pr-size-labeler@v1
+        with:
+          xs_label: 'size/xs'
+          xs_max_size: '10'
+          s_label: 'size/s'
+          s_max_size: '100'
+          m_label: 'size/m'
+          m_max_size: '500'
+          l_label: 'size/l'
+          l_max_size: '1000'
+          xl_label: 'size/xl'
+          fail_if_xl: 'false'
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          files_to_ignore: 'package-lock.json yarn.lock go.sum'
```

I wonder if this will work. 

```suggestion
          files_to_ignore: >
            package-lock.json
            yarn.lock
            go.sum
```